site/app/controllers/grading/SimpleGraderController.php

```diff
@@ -172,7 +172,7 @@ public function gradePage($gradeable_id, $view = null, $sort = null) {
             $graders[$section->getName()] = $section->getGraders();
         }
 
-        $rows = $this->core->getQueries()->getGradedGradeables([$gradeable], $student_ids, null, [$section_key, $sort_key]);
+        $rows = $this->core->getQueries()->getGradedGradeables([$gradeable], $student_ids, null, [$section_key, "u.registration_subsection", $sort_key]);
```

Comment:
We should only include this clause if `$section_key` is for registration sections.

Reply:
This was put in as a temporary change, and we plan on allowing the grader to choose whether they want to sort by subsection in the future.